handsontable/src/core.js

```diff
@@ -50,13 +50,35 @@ let activeGuid = null;
  *
  * ## How to call a method
  *
+ * ::: only-for javascript
  * ```js
  * // First, let's contruct Handsontable
  * const hot = new Handsontable(document.getElementById('example'), options);
  *
  * // Then, let's use the setDataAtCell method
  * hot.setDataAtCell(0, 0, 'new value');
  * ```
+ * :::
+ *
+ * ::: only-for react
+ * ```jsx
+ * const hotRef = useRef();
+ *
+ * ...
+ *
+ * // First, let's contruct Handsontable
+ * <HotTable
+ *   ref={hotRef}
+ *   settings={options}
+ * />
+ *
+ * ...
+ *
+ * const hot = hotRef.current.hotInstance;
+ * // Then, let's use the setDataAtCell method
+ * hot.setDataAtCell(0, 0, 'new value');
+ * ```
```

Comment:
@wszymanski – as per [this comment](https://handsoncode.slack.com/archives/C034WPQUG9W/p1661169136287979?thread_ts=1661165254.475329&cid=C034WPQUG9W), let's add this bit here (only for React):

```
Related guides:
- [Instance methods](@/guides/getting-started/react-methods.md)
```